components/brave_referrals/browser/BUILD.gn

```diff
@@ -16,11 +16,9 @@ static_library("browser") {
 
     deps = [
       "//base",
-      "//brave/common",
       "//brave/components/brave_referrals/common",
       "//brave/components/brave_stats/browser",
       "//brave/vendor/brave_base",
-      "//chrome/common",
       "//components/prefs",
       "//content/public/browser",
       "//net",
```

Comment:
where's the dep?

Reply:
Fixed
We can't remove `check_includes = false` because the component uses `//chrome/browser` ;(